docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+
+## Capabilities and benefits
+
+[[= product_name_headless =]] is a tool box with a Back Office.
+It comes without a default front office.
+You don't lose time to develop a theme for a provided front office before discovering it won't fit your needs.
+No distraction.
+[[= product_name_headless =]] helps you focus on the content, create and organize with its straightforward user interface (UI),
+imagine its inputs/outputs, and implement them with its various layers' APIs.
+
+### Core features
+
+The core of [[= product_name =]] Headless edition is already awesome as it offers everything to structure your content repositories and access them.
+
+#### Content model
+
+- Content items are organized as a tree in a repository.
+- An item can have multiple locations in this tree.
+- Content items are typed.
+- Content types are sets of typed data fields, with eventually conditions on the possible values.
+- Rich Text Field Type comes with an online editor. See [Online Editor product guide](online_editor_guide.md) for more.
+- Multilingual, it can store a content in several languages, the content model define which field must be translated, and which don't vary.
+
+See [Content management product guide](content_management_guide.md) for more.
+
+#### User management
+
+- User and user group rights are set by roles with thin granular limited permission policies in a safe deny-by-default security system.
+- Users are content items as well, so your knowledge about content management is reused.
+
+See [User management product guide](user_management_guide.md) for more.
+
+#### Content access
+
+- The REST API and GraphQL API support access to and edition of the content.
+- [[= product_name_headless =]] offers a complete PHP API to extend the ways to access content.
+- A design engine and a view controller offer to create plain text content views (such as HTML, JSON, XML, CSS, JS, CSV, or Markdown),
+  and to factorize those views by using theme cascades.
+  This design engine is used in the Back Office which is equally extendable.
+- Multichannel, content can be accessed through several channel configurations,
+  such as the domain name it replies to, the sub-part of the content tree it starts from, the users rights, or the design theme.
+  The Back Office itself is such a channel.
+- Multi-repository, the same platform can use separate databases if impermeability is needed between channel groups.
+
+### Advanced features
+
+On top of this strong core, [[= product_name =]] Headless edition brings tools to increase user experience, from final front users to Back Office contributors.
+
+#### [[= product_name_headless =]] is a complete Digital Experience Platform (DXP)
+
+- [[= product_name_headless =]] comes with a personalization engine, allowing to recommend content to the front user according to its behavior,
+  or, when authenticated, contents matching user's segment/group. See [Personalization product guide](personalization_brochure.md) for more.
```

Comment:
```suggestion
- ```suggestion
- [Personalization](personalization_brochure.md) engine, which allows you recommend content to end users according to their behavior,
  or, when authenticated, by matching with their segment/group.
```